.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discussions
+    url: https://github.com/streetcomplete/StreetComplete/discussions
+    about: For discussions about the usage of StreetComplete, asking questions or talking about visions.
```

Comment:
```suggestion
    about: For discussions about the usage of StreetComplete, asking questions or talking about ideas which are not yet actionable.
```

@matkoniecz original is probably slightly clearer, although it's very easy to have a "specific proposed change" which can't actually be actioned because there are some edge cases the tagging doesn't handle (which might not have occurred to the reporter). Are they supposed to sit in discussions until a tagging proposal is resolved or? i.e. how concrete an idea are we trying to target here? Picking a tag literally at random which we don't currently support (incidentally quite hard), gives me `location=`; I could muddle together a quest proposal with a specific proposed change to tag this for everything or some items, but it's likely to have a lot of edge cases. Are we trying to keep that as currently, or mull it over in discussions until the problems are resolved? Or simply stop vague ideas like "Add Augmented Reality quest view" being raised?

Personally I'm not a huge fan of the discussion UI in terms of seeing who's commented/what's changed but maybe others don't find it so bad?

Reply:
> can't actually be actioned because there are some edge cases the tagging doesn't handle (which might not have occurred to the reporter)

in this case issue is also fine, it will be simply WONTFIXed

Reply:
Strange that GitHub didn't credit you [peternewman](https://github.com/peternewman) although I committed your suggestion. Instead it was attributed to [matkoniecz](https://github.com/matkoniecz).